data/map.txt

```diff
@@ -30782,6 +32481,24 @@ planet Ochrescoop
 	bribe 0.1
 	security 0.5
 
+planet Okoity
+	attributes bunrodea urban
+	landscape land/fog2
+	description `Okoity has the highest population of any Bunrodean world, with a staggering 24 billion people. All but the most inhospitable regions of the planet have been settled in some shape or form.`
+	description `	The planet saw a large influx of people with the destruction of Olohoa's atmosphere in the nearby system of Sera Natta, massive even relative to the already preexisting population. Although this occurred about a century ago, the planet still seems to be struggling with this increased population.`
+	spaceport `The spaceport of Okoity is completely packed with Bunrodeans. You have little room to maneuver as you explore the spaceport while trying to not bump into anyone. Many of the Bunrodeans here show the signs of being genetically engineered, from Bunrodeans with extra limbs to one that you see up high climbing on walls and jumping from building to building to avoid the crowd below.`
+	shipyard "Bunrodea Basic"
+	outfitter "Bunrodea Basic"
+	"required reputation" 25
+	bribe 0
+	security 0
+
+planet Olohoa
+	attributes bunrodea uninhabited
+	landscape land/sky3
+	description `From afar, Olohoa looks like it might be a pleasant place to live, with clear oceans not too disimilar to what you might find on a Paradise World. But your ship's sensors tell you a different story, one of a planet with immense amounts of ultraviolet radiation bombarding its surface. Nothing short of an extremophile could survive on the surface here, and any life in the oceans is likely deep down where no radiation can reach.`
+	description `	As you explore, you begin to find occasional signs of civilization in the form of small cities with landing pads for ships, but these areas are just as lifeless as the rest of the planet.`
```

Comment:
Making this system just naturally barren

```suggestion
	description `From afar, Olohoa looks like it might be a pleasant place to live, with clear oceans not too disimilar to what you might find on a Paradise World, were you to ignore the local star. But your ship's sensors tell you a different story, one of a planet with immense amounts of ultraviolet radiation bombarding its surface. Nothing short of an extremophile could survive on the surface here, and any life in the oceans is likely deep down where no radiation can reach.`
```